src/api.py

```diff
@@ -0,0 +1,44 @@
+import hashlib
+import requests
```

Comment:
Can you add an empty line to separate standard import (e.g. `hashlib`) from external code (`requests`)?

Reply:
We still like to have imports in three groups:

1. standard library 
2. one empty line
3. external third-part libraries
4. one empty line
5. own module imports
6. two empty lines

And each block of import are then sorted.
So here that would mean:
```
import os
import tempfile 

import requests


<... rest of the code>
```